src/Cake/Scripting/Roslyn/RoslynScriptEngine.cs

```diff
@@ -11,19 +12,21 @@ namespace Cake.Scripting.Roslyn
     internal sealed class RoslynScriptEngine : IScriptEngine
     {
         private readonly CakeOptions _options;
+        private readonly ICakeConfiguration _configuration;
         private readonly IAssemblyLoader _loader;
         private readonly ICakeLog _log;
 
-        public RoslynScriptEngine(CakeOptions options, IAssemblyLoader loader, ICakeLog log)
+        public RoslynScriptEngine(CakeOptions options, ICakeConfiguration configuration, IAssemblyLoader loader, ICakeLog log)
```

Comment:
@patriksvensson if I remember correctly, you where removing places where we pass `IConfiguration` around in your CLI rewrite PR?

Reply:
Yes this part is not needed when using the CLI review RFC0001.  I have also made the changes for the cache using the RFC0001 pull request.